proxy/src/main/resources/log4j2.xml

```diff
@@ -22,18 +22,18 @@
     <TerminalConsole name="TerminalConsole">
       <PatternLayout>
         <LoggerNamePatternSelector
-          defaultPattern="%highlightError{[%d{HH:mm:ss} %level] [%logger]: %minecraftFormatting{%msg}%n%xEx}">
+          defaultPattern="%highlightError{[%d{HH:mm:ss} %level] [%logger]: %msg%n%xEx}">
           <!-- Velocity doesn't need a prefix -->
           <PatternMatch key="com.velocitypowered."
-            pattern="%highlightError{[%d{HH:mm:ss} %level]: %minecraftFormatting{%msg}%n%xEx}"/>
+            pattern="%highlightError{[%d{HH:mm:ss} %level]: %msg%n%xEx}"/>
         </LoggerNamePatternSelector>
       </PatternLayout>
     </TerminalConsole>
     <RollingRandomAccessFile name="File" fileName="logs/latest.log"
       filePattern="logs/%d{yyyy-MM-dd}-%i.log.gz"
       immediateFlush="false">
       <PatternLayout
-        pattern="[%d{HH:mm:ss}] [%t/%level] [%logger]: %minecraftFormatting{%msg}{strip}%n"/>
+        pattern="[%d{HH:mm:ss}] [%t/%level] [%logger]: %msg%n"/>
```

Comment:
What did the `{strip}` do here?

Reply:
Removes ansi color formatting when saving logs. https://github.com/Minecrell/TerminalConsoleAppender/blob/master/src/main/java/net/minecrell/terminalconsole/MinecraftFormattingConverter.java#L126

Reply:
The [corresponding paper patch](https://github.com/PaperMC/Paper/pull/9313/files#diff-ef193170056b3e41ea067f8e473d2c7e675b85ef9ee65f2f8cd94b65152f6ba9L61-R61) added a [`stripAnsi`](https://github.com/PaperMC/Paper/pull/9313/files#diff-6c1396d60730e7053f0b761bdb487752467c9bc0444a4aac41908376b38a56bdR198-R233) processor. Is that not required here?

Reply:
Of course, yesterday after 1 year I was trying again to make a Log4j2 plugin for Velocity but this time to remove the ansi codes, but it is still being impossible because it is not being recognized as it was last year https://github.com/PaperMC/Velocity/pull/718#issuecomment-1226019613

This processor is necessary even without this pull request because the minecraftFormatting plugin only avoids adding the codes that were converted from the legacy format, it does not remove the ones that have already been implemented. I will keep trying to fix that, but anyone is free to implement a workaround if they can.

Reply:
Thanks for the clarification, I wasn't aware of any earlier struggles, but good luck

Reply:
implemented